### PR TITLE
Set listener once

### PR DESCRIPTION
**Pull request by kesha-antonov**
#### What's this PR do?

Set listener only once. Not on each "on".

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,30 +58,29 @@ const MusicControl = {
     }
   },
   on: function(actionName, cb){
-    if(subscription){
-      subscription.remove();
+    if(listenerOfNativeMusicControl == null){
```
Comment (tanguyantoine) [inline index.js]: What about :
```js
if (!Boolean(listenerOfNativeMusicControl)){
``` 
or
```js
if (!listenerOfNativeMusicControl) {
```

File: index.js [hunk 2]
```diff
@@ -58,30 +58,29 @@ const MusicControl = {
...
-        }
-      );
     handlers[actionName] = cb
   },
   off: function(actionName, cb){
     delete(handlers[actionName])
-    if(!Object.keys(handlers).length && subscription){
-      subscription.remove()
-      subscription = null;
+    if(!Object.keys(handlers).length && listenerOfNativeMusicControl != null){
```
Comment (tanguyantoine) [inline index.js]: same

File: index.js [hunk 3]
```diff
@@ -58,30 +58,29 @@ const MusicControl = {
...
+      listenerOfNativeMusicControl.remove()
+      listenerOfNativeMusicControl = null;
     }
   },
   stopControl: function() {
-    if (subscription) {
-      subscription.remove();
+    if (listenerOfNativeMusicControl != null) {
```
Comment (tanguyantoine) [inline index.js]: same

File: index.js (hunk 1 above)
Comment (Guichaguri) [reply index.js]: The second one is better, it also handles `undefined`

Comment (tanguyantoine) [reply index.js]: the first as well ;-)

**Comment by kesha-antonov** [comment]
I think it's not sufficient since ```!= null``` check for null and undefined. But anyway. Changed.

**Comment by tanguyantoine** [comment]
Thank you

**Comment by tanguyantoine** [comment]
published as 0.5.4

**Comment by kesha-antonov** [comment]
Thanks!

**Pull request merged**